meinberlin/apps/offlineevents/templates/meinberlin_offlineevents/offlineevent_detail.html

```diff
@@ -20,4 +20,4 @@ <h2 class="item-detail__title">{{ object.name }}</h2>
             </article>
         </div>
     </div>
-{% endblock %}
+{% endblock %} -->
```

Comment:
added styling from the offlineevent_detail.html template which appears not to be used, have commented out for now for testing